intro/webhooks_intro.md

```diff
@@ -202,7 +213,7 @@ For Creative Cloud Asset events, you&rsquo;ll need to add the Creative Cloud Lib
 
 Your webhook URL must by necessity be accessible from the open internet. This means third-party actors can send forged requests to it, tricking your application into handling fake events.
  
-To prevent this from happening, Adobe I/O Events will add a `x-adobe-signature` header to each POST request it does to your webhook URL, which allows you to verify that the request was really made by Adobe I/O Events.
+To prevent this from happening, Adobe I/O Events will add a `x-adobe-signature` header to each Http request it does to your webhook URL, which allows you to verify that the request was really made by Adobe I/O Events.
```

Comment:
... each **HTTP request it sends** to...

Reply:
done https://github.com/AdobeDocs/adobeio-events/pull/77/commits/bfbbd5075a485a59bc83a5b4963e7d8c779e9a52